lib/punch_clock.rb

```diff
@@ -5,21 +5,8 @@ class PunchClock
 
   VERSION_NAME = "The Baddest Man on the Planet"
 
-  MIDNIGHT_MADNESS_NOTES = [
-    "Get some sleep!",
-    "Don't you have any hobbies?",
-    "Get some rest, (wo)man...",
-    "You should go to bed.",
-    "That can't be healthy.",
-    "You might need therapy.",
-    "All work and no play makes Jack a dull boy.",
-    "You need to get your priorities straight.",
-    "Work-life balance. Ever heard of it?",
-    "Did you know that the average adult needs 7-8 hours of sleep?"
-  ].freeze
-
```

Comment:
Why did they have to go? They were too young :sob:

Reply:
You're right. They don't deserve to die this early just because I was too lazy to port it to the multi day approach. Will redeem it.